changelog.txt

```diff
@@ -4,6 +4,15 @@ Changes
 
 The **signac-flow** package follows `semantic versioning <https://semver.org/>`_.
 
+next
+----
+
+Changed
+++++++
+
+- Status updates only pull from the cache once rather than once per operation.
```

Comment:
Could you rephrase along the lines of "The performance of X has been significantly improved (factor X) by optimizing Y"?

Btw, no need to skip CI tests.

Reply:
And add the PR # in parentheses.